forward43/scrape_some_good.py

```diff
@@ -0,0 +1,65 @@
+import json
+import requests
+from bs4 import BeautifulSoup
+import pandas as pd
+
+# create url's for ... overview pages
+browse_url_base        = 'https://startsomegood.com/projects?page='
+browse_pages = []
+
+for page in range(15):
+    browse_pages.append(browse_url_base + str(page))
+
+# create dataframe to store data
+column_names            = ["Title", "Description", "Status", "innovation_type", "Country", "City", "Contact person/details", "Link"]
+good_data               = []
+
+# loop over the overview pages to find the project tiles
+project_url_base        = 'https://startsomegood.com'
+
+for URL in browse_pages:
+    RESPONSE_OVERVIEW = requests.get(URL, timeout = 5)
```

Comment:
This line can potentially raise exceptions. I'd recommend wrapping it around a try except block.